File: DAY2/원정/방_번호_1475.py
import math
n =input()

dic = {}

for i in range(0, 10):
    dic[i] = 0
    
for i in n:
    number = int(i)
    if number == 6 or number == 9:
        dic[6] += 1
    else:
        dic[number] += 1
    
dic[6] = math.ceil(dic[6] / 2)

max = 0
for i in dic:
    if max < dic[i]:
        max = dic[i]

print(max)

File: DAY2/성욱/방번호 - 1475.js
const input = require('fs')
  .readFileSync('/dev/stdin')
  .toString()
  .trim()

const numbers = new Array(9).fill(0)

for (let i = 0; i < input.length; i++) {
	if (input[i] === '9') {
		numbers[6]++;
	} else {
		numbers[Number(input[i])]++
	}
}

numbers[6] = Math.ceil(numbers[6] / 2)

const max = Math.max(...numbers)

console.log(max)